tests/drivers/dma/chan_link_transfer/boards/mimxrt1024_evk.overlay

```diff
@@ -0,0 +1 @@
+dma0: &edma0 { };
```

Comment:
would probably be better to change the test to use an alias instead of an arbitrary nodelabel, and add that alias to the board dts. This is a good time to do it because this test is only supported by a couple boards that would need this change, better sooner than later